### ISSUE TITLE
Legal and Lorem snippets should be removable/editable

### ISSUE THREAD

**Issue by tjkohli** [atom/language-text#11]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Legal and lorem snippets appear at the top of the list for every suggestion starting with "l"

### Steps to Reproduce

1. Open html or css or sass file. Type "l"

**Expected behavior:** [What you expect to happen]

Stupid legal and lorem suggestions do not appear.

**Actual behavior:** [What actually happens]

Stupid legal and lorem suggestions appear at the top of the list.

**Reproduces how often:** [What percentage of the time does it reproduce?]

100%.

### Versions

You can get this information from copy and pasting the output of `atom --version` and `apm --version` from the command line. Also, please include the OS and what version of the OS you're running.

Atom    : 1.12.9
Electron: 1.3.13
Chrome  : 52.0.2743.82
Node    : 6.5.0

apm  1.12.9
npm  3.10.5
node 4.4.5
python 2.7.10
git 2.10.1

macOS 10.12.2

### Additional Information

I tried everything, dug into the .atom folder, tried overriding it in my snippets.cson, but these won't go away and they drive me crazy, especially whenever I type "left" in CSS.

![screen shot 2017-01-09 at 1 49 01 pm](https://cloud.githubusercontent.com/assets/7361247/21778769/c17092c8-d672-11e6-9f4a-08d5aa8d7425.png)

**Comment by clsmt** [comment]
I agree with you totally. I have to disable the language-text package entirely just to disable lorem and legal. They pop up everytime I type any word starting with l. 
Please disable these two snippets.

**Comment by scherii** [comment]
https://github.com/atom/snippets/issues/55
